src/main/java/seedu/address/model/contact/ContactComparatorByName.java

```diff
@@ -17,7 +17,7 @@
      */
     @Override
     public int compare(Contact contact, Contact otherContact) {
-        int value = contact.getName().fullName.compareToIgnoreCase(otherContact.getName().fullName);
+        int value = contact.getName().getName().compareToIgnoreCase(otherContact.getName().getName());
         if (value > 0) {
```

Comment:
As discussed beofore, are we going to remove `Contact#getName()` and use `Contact#toString()` instead?

Reply:
Yes, this will be modified in a separate PR.